packages/kbn-search-api-panels/components/preprocess_data.tsx

```diff
@@ -0,0 +1,228 @@
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFlexGroup alignItems="flexStart" justifyContent="flexStart">
+              <EuiFlexItem grow={false}>
+                <EuiToken iconType="logstashFilter" size="l" color="#466B8D" />
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiTitle size="s">
+                  <h3>
+                    {i18n.translate('searchApiPanels.pipeline.overview.dataFiltering.title', {
+                      defaultMessage: 'Data filtering',
+                    })}
+                  </h3>
+                </EuiTitle>
+                <EuiSpacer size="s" />
+                <EuiText size="s">
+                  {i18n.translate('searchApiPanels.pipeline.overview.dataFiltering.description', {
+                    defaultMessage:
+                      'Remove fields from documents like sensitive information from documents before indexing.',
+                  })}
+                </EuiText>
+                <EuiSpacer size="s" />
+                <EuiText>
+                  <p>
+                    <EuiLink href={'#'} target="_blank">
+                      {i18n.translate(
+                        'searchApiPanels.preprocessData.overview.dataFiltering.learnMore',
+                        {
+                          defaultMessage: 'Learn More',
+                        }
+                      )}
+                    </EuiLink>
+                  </p>
+                </EuiText>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFlexGroup alignItems="flexStart" justifyContent="flexStart">
+              <EuiFlexItem grow={false}>
+                <EuiToken iconType="tokenArray" size="l" fill="none" color="#466B8D" />
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiTitle size="s">
+                  <h3>
+                    {i18n.translate('searchApiPanels.pipeline.overview.arrayJsonHandling.title', {
+                      defaultMessage: 'Array/JSON handling',
+                    })}
+                  </h3>
+                </EuiTitle>
+                <EuiSpacer size="s" />
+                <EuiText size="s">
+                  {i18n.translate(
+                    'searchApiPanels.pipeline.overview.arrayJsonHandling.description',
+                    {
+                      defaultMessage: 'Run batch processors,  parse JSON data and sort elements.',
+                    }
+                  )}
+                </EuiText>
+                <EuiSpacer size="s" />
+                <EuiText>
+                  <p>
+                    <EuiLink href={'#'} target="_blank">
+                      {i18n.translate(
+                        'searchApiPanels.preprocessData.overview.arrayJsonHandling.learnMore',
+                        {
+                          defaultMessage: 'Learn More',
```

Comment:
```suggestion
                          defaultMessage: 'Learn more',
```

idem for all instances